src/EditorFeatures/Text/TextEditorFeatures.csproj

```diff
@@ -58,6 +58,8 @@
     <InternalsVisibleToTest Include="Roslyn.Services.Editor.VisualBasic.UnitTests" />
     <InternalsVisibleToTest Include="Roslyn.VisualStudio.Services.UnitTests" />
     <InternalsVisibleToTest Include="RoslynETAHost" />
+    <InternalsVisibleToFSharp Include="FSharp.Editor" />
```

Comment:
You mean naming the F# dlls? Currently these are the names of the existing F# LS dlls. @otawfik-ms's  plan is to start adding code to those existing dlls and refactor later.